our-process/policies.md

```diff
@@ -44,16 +44,35 @@ already published on `PyPI` or `conda-forge`.
 
 ### Publication with Journal of Open Source Software (JOSS)
 
-If you have previously published your software package with JOSS, you can still
-submit it to pyOpenSci for review. This provides:
+If you have previously published your software package with the Journal of Open
+Source Software (JOSS), you can still submit it to pyOpenSci for review. This
+provides increased visibility for your package as a vetted tool within the scientific
+Python ecosystem and access to our long-term maintenance support.
 
-- Increased visibility of your package as a vetted tool within the scientific Python
-  ecosystem
-- We will also keep in touch with you as a maintainer to support long-term
-  maintenance. If you need to step down from maintaining your package, we will help
-  find a new maintainer and/or help sunset the tool.
+Since your package has already undergone a JOSS review, we have a specific,
+expedited review process to streamline the submission process and save time.
+
+#### Expedited Review Process
+
+We offer two pathways for packages previously reviewed by JOSS:
+
+1. Fast-Track Review (Editor-Only):
+   Your package is eligible for this fast-track review if it has been published by
+   JOSS within the last year and has not had a major version release (e.g., v1.x to v2.0)
+   since its JOSS publication. In this case, an editor will conduct the review by going
+   through our pyOpenSci submission checklist to ensure all our specific requirements are met.
+
+2. Expedited Review (Editor + One Reviewer):
+   If your package's JOSS publication is over a year old or if it has had a major version release
+   since its JOSS publication, it will undergo an expedited review with one editor and one external reviewer.
+   The editor and reviewer will focus on any significant changes and ensure the package meets all current pyOpenSci
+   standards. This approach reduces the burden of a full review while ensuring the quality of the package
+   reflects its most recent version.
+
+We will also keep in touch with you as a maintainer to support long-term maintenance. If you need to step down from maintaining your package, we will help find a new maintainer and/or help sunset the tool.
```

Comment:
> We will also keep in touch with you as a maintainer to support long-term maintenance.

Isn't this rather general? Or is this specific to software _also_ published in JOSS?

Reply:
I had this thought too. Is this the case for any packages reviewed by pyopensci?